### DISCUSSION TITLE
ReactiveUI and Uno

### DISCUSSION THREAD

**Discussion by HavenDV** [unoplatform/uno#9168] [General]
Since the [ReactiveUI.Uno repository seems to be abandoned](https://github.com/reactiveui/ReactiveUI.Uno) (and the ReactiveUI.Uno package is not released with new versions), is there any plan to revive the [Uno.ReactiveUI](https://www.nuget.org/packages/Uno.ReactiveUI) package?

**Comment by jeromelaban** [comment]
We will not be supporting the `Uno.Reactive` repo again. The reason why the ReactiveUI.Uno package was removed was because of the weird state of .NET 6 Mobile before the MAUI GA release. [This PR should be reverted](https://github.com/reactiveui/ReactiveUI/pull/3212) to get the packages back.
